index.html

```diff
@@ -237,20 +237,20 @@ <h2>Layer controls</h2>
 			<button class="accordion">Coherent network</button>
 			<div class="panel">
 				<p>Indicates a suggested order in which the network should grow.</p>
-				<p><label><input type="checkbox" class="showlayer" data-layer="cohesivenetwork" value="false"> Show cohesive network</label></p>
-				<div id="cohesivenetworklegend" class="legend"></div>
+				<p><label><input type="checkbox" class="showlayer" data-layer="coherentnetwork" value="false"> Show coherent network</label></p>
+				<div id="coherentnetworklegend" class="legend"></div>
 			</div>
 			
-			<button class="accordion">Cycling by design compliance</button>
+			<button class="accordion">Cycling by Design compliance</button>
```

Comment:
:+1: although I wonder: should we drop the rather superfluous "compliance" bit:

```suggestion
			<button class="accordion">Cycling by Design</button>
```

Reply:
Well I think the point is that the data is intended to show the level of compliance compared with CbD. So compliance presumably necessary.

Reply:
The thing is: it shows lots of other things also, not just compliance. But happy with it as is for now.

Reply:
Yes, fair point.